code/modules/shuttle/gamemodes/crash.dm

```diff
@@ -3,46 +3,45 @@
 	id = "canterbury_dock"
 	name = "Canterbury Crash Site"
 	dir = SOUTH
-	width = 15
-	height = 24
-	dwidth = 7
-	dheight = 12
-
+	width = 19
+	height = 31
+	dwidth = 9
+	dheight = 15
 
-/obj/docking_port/stationary/crashmode/on_crash()
+///obj/docking_port/stationary/crashmode/on_crash()
 	//clear areas around the shuttle with explosions
-	var/turf/C = return_center_turf()
-
```

Comment:
Fully delete this instead of commenting it out

Reply:
Done as requested.